packages/communication-react/src/mergedHooks.ts

```diff
@@ -5,14 +5,16 @@ import {
   getCallingSelector,
   GetCallingSelector,
   useCallingHandlers,
-  useCallingSelector
+  useCallingSelector,
```

Comment:
make sure to test if the treeshaking works as expected, sometimes if an import like this one is used in the same file treeshaking fails to get rid of it. if so we'd need to move the new fns to their own file